### DISCUSSION TITLE
How to filter traces by multiple values

### DISCUSSION THREAD

**Discussion by DarioBernardo** [SigNoz/signoz#5830] [Q&A]
I'm trying to create a dashboard in the SigNoz web interface and I'm having trouble setting up a filter to include multiple email domains. Here's what I've tried so far:

Current setup:
- I've created a 'value' panel in the dashboard
- In the query builder, i have selected 'traces'  and added a few filter conditions, like `service.name = myservice`
- I can filter for a single domain with `user.email CONTAINS @mydomain`

Problem:
I want to extend the filter to include multiple email domains, but none of my attempts have worked. Here are some of the variations I've tried:

1. `user.email CONTAINS @mydomain OR user.email CONTAINS @myotherdomain`
2. `user.email IN (@mydomain, @myotherdomain)`
3. `user.email =~ "@(mydomain|myotherdomain)"`
4. `user.email CONTAINS @mydomain AND NOT user.email CONTAINS @myotherdomain`
5. `user.email = "*@mydomain" OR user.email = "*@myotherdomain"`

None of these syntaxes seem to work in SigNoz. The filter either doesn't apply at all, or it doesn't return the expected results.

Questions:
 What is the correct syntax in SigNoz to filter traces by multiple email domains?

Any help or guidance would be greatly appreciated. If you need any additional information about my setup or what I've tried, please let me know.

**Comment by zeusssz** [comment]
unless im mistaken, you may need to add the tld at the end of this
> user.email =~ "@(mydomain|myotherdomain)"

**Reply by DarioBernardo** [reply]
I am not sure I understand, can you show what you mean? Also, i don't think =~ is an accepted operation. I am quite sure changing the string won't work, but happy to try anything.

**Reply by zeusssz** [reply]
im rereading the docs, i am unsure now as well.
